exchanges/btse/btse_types.go

```diff
@@ -117,52 +143,106 @@ type MarketStatistics struct {
 // ServerTime stores the server time data
 type ServerTime struct {
 	ISO   time.Time `json:"iso"`
-	Epoch string    `json:"epoch"`
+	Epoch int       `json:"epoch"`
 }
 
 // CurrencyBalance stores the account info data
 type CurrencyBalance struct {
 	Currency  string  `json:"currency"`
-	Total     float64 `json:"total,string"`
-	Available float64 `json:"available,string"`
+	Total     float64 `json:"total"`
+	Available float64 `json:"available"`
 }
 
-// Order stores the order info
-type Order struct {
-	ID        string  `json:"id"`
-	Type      string  `json:"type"`
-	Side      string  `json:"side"`
-	Price     float64 `json:"price"`
-	Amount    float64 `json:"amount"`
-	Tag       string  `json:"tag"`
-	Symbol    string  `json:"symbol"`
-	CreatedAt string  `json:"created_at"`
+// AccountFees stores fee for each currency pair
+type AccountFees struct {
+	MakerFee float64 `json:"makerFee"`
+	Symbol   string  `json:"symbol"`
+	TakerFee float64 `json:"takerFee"`
 }
 
-// OpenOrder stores an open order info
-type OpenOrder struct {
-	Order
-	Status string `json:"status"`
+// TradeHistory stores user trades for exchange
+type TradeHistory []struct {
+	Base         string  `json:"base"`
+	ClOrderID    string  `json:"clOrderID"`
+	FeeAmount    float64 `json:"feeAmount"`
+	FeeCurrency  string  `json:"feeCurrency"`
+	FilledPrice  float64 `json:"filledPrice"`
+	FilledSize   float64 `json:"filledSize"`
+	OrderID      string  `json:"orderId"`
+	OrderType    int     `json:"orderType"`
+	Price        float64 `json:"price"`
+	Quote        string  `json:"quote"`
+	RealizedPnl  int     `json:"realizedPnl"`
+	SerialID     int     `json:"serialId"`
+	Side         string  `json:"side"`
+	Size         float64 `json:"size"`
+	Symbol       string  `json:"symbol"`
+	Timestamp    string  `json:"timestamp"`
+	Total        int     `json:"total"`
+	TradeID      string  `json:"tradeId"`
+	TriggerPrice float64 `json:"triggerPrice"`
+	TriggerType  int     `json:"triggerType"`
+	Username     string  `json:"username"`
+	Wallet       string  `json:"wallet"`
+}
+
+// WalletHistory stores account funding history
+type WalletHistory []struct {
+	Amount      float64 `json:"amount"`
+	Currency    string  `json:"currency"`
+	Description string  `json:"description"`
+	Fees        float64 `json:"fees"`
+	OrderID     int64   `json:"orderId"`
+	Status      int     `json:"status"`
+	Timestamp   int64   `json:"timestamp"`
+	Type        int     `json:"type"`
```

Comment:
Type -> WalletType or somethn

Reply:
also same thing is there in data structs: Trade and wsOrderUpdate just good if u change that too pls

Reply:
bump